src/hooks/useInitialValues.ts

```diff
@@ -60,22 +60,21 @@ export function useInitialValues(
     }
     if (encounter) {
       formFields
-        .filter((field) => isEmpty(field.value))
+        .filter((field) => isEmpty(field.meta?.previousValue))
         .filter((field) => field.questionOptions.rendering !== 'file')
         .forEach((field) => {
           if (hasRendering(field, 'repeating') && !field.meta?.repeat?.isClone) {
             repeatableFields.push(field);
-            if (field.type === 'obsGroup') {
-              return;
-            }
           }
           let existingVal = formFieldHandlers[field.type]?.getInitialValue(
             encounter,
             field,
             formFields,
             encounterContext,
           );
-
+          if (field.type === 'obsGroup') {
+            return;
+          }
```

Comment:
We let the obs-group owner or parent get initialised like other fields but since it's value will always be undefined, don't bother initialising it in the values map.